x/staking/README.md

```diff
@@ -690,9 +738,15 @@ This message is expected to fail if:
 * the source validator has a receiving redelegation which is not matured (aka. the redelegation may be transitive)
 * existing `Redelegation` has maximum entries as defined by `params.MaxEntries`
 * the `Amount` `Coin` has a denomination different than one defined by `params.BondDenom`
+* the delegation is a `ValidatorBond` and the reduction in validator bond would cause the existing liquid delegation to exceed the cap.
+* the delegator is a liquid staking provider and the delegation exceeds
+either the `GlobalLiquidStakingCap`, the `ValidatorLiquidStakingCap` or the validator bond cap.
 
 When this message is processed the following actions occur:
 
+* if the delegation if is a validator bond, the `ValidatorBondShares` of the validator is decreased.
+* if the delegator is a liquid staking provider, the `TotalLiquidStakedTokens`` increased
+and the validator's `LiquidShares` is decreased.
```

Comment:
Differentiate between the two validators.

Reply:
Good catch.